app/helpers/cross_domain_analytics_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CrossDomainAnalyticsHelper
+  def cross_domain_tracker(slug)
+    {
+      'register-to-vote' => 'UA-23066786-5',
+      'accelerated-possession-eviction' => 'UA-37377084-12',
+      'renewtaxcredits' => 'UA-43414424-1', # HMRC, https://www.pivotaltracker.com/n/projects/1261204
```

Comment:
While the link is available, it is useful. Link rot for the history and context of our backlog is the problem that ought to be fixed. Not having a permanent link back to context/discussion for any given story makes excavation more difficult, even if commits themselves are descriptive and useful.